examples/leaflet_local/large.html

```diff
@@ -0,0 +1,145 @@
+    <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"></script>
+    <script src="/dist/flatgeobuf-geojson.min.js"></script>
+    <script src="https://unpkg.com/json-formatter-js"></script>
+
+    <style>
+        #map { height: 480px; }
+    </style>
+</head>
+<body>
+    <ul class="primary-navigation">
+        <li class="active">
+            Leaflet Example
+        </li>
+        <li>
+            <a href="/examples/openlayers_local/">OpenLayers Example</a>
+        </li>
+    </ul>
+    <ul class="secondary-navigation">
+        <li><a href="/examples/leaflet_local/">Basic Example</a></li>
+        <li><a href="/examples/leaflet_local/filtered.html">Filter By Rect</a></li>
+        <li class="active">Filtering a Large Dataset</li>
+    </ul>
+
+    <div id="map"></div>
+    <script>
+        document.addEventListener("DOMContentLoaded", async () => { 
+            // basic OSM Leaflet map
+            let map = L.map('map').setView([40.766, -73.98], 14);
+            L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+                maxZoom: 19,
+                attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+            }).addTo(map);
+
+            // optionally show some meta-data about the FGB file
+            function handleHeaderMeta(headerMeta) {
+                const header = document.getElementById('header')
+                const formatter = new JSONFormatter(headerMeta, 10)
+                header.appendChild(formatter.render())
+            }
+
+            // For the example, we fix a visible Rect in the middle of the map
+            function getBoundForRect() {
+                const widthMeters = 2000;
+                return map.getCenter().toBounds(widthMeters);
+			}
+
+            // convert the rect into the format flatgeobuf expects
+            function fgBoundingBox() {
+				const bounds = getBoundForRect();
+				return {
+					minX: bounds.getWest(),
+                    maxX: bounds.getEast(),
+                    minY: bounds.getSouth(),
+                    maxY: bounds.getNorth(),
+                };
+            }
+
+            // track the previous results so we can remove them when adding new results
+            let previousResults = L.layerGroup().addTo(map);
+            async function updateResults() {
+                // remove the old results
+                previousResults.remove();
+                const nextResults = L.layerGroup().addTo(map);
+                previousResults = nextResults;
+
+                // Use flatgeobuf JavaScript API to iterate features as geojson.
+                // Because we specify a bounding box, flatgeobuf will only fetch the relevant subset of data,
+                // rather than the entire file.
+                let iter = flatgeobuf.deserialize('https://s3.amazonaws.com/mjk_asdf/abs/population_areas.fgb', fgBoundingBox(), handleHeaderMeta);
```

Comment:
This is my go-to 12.3GB FGB of United State census tracts. Is there an "official" FGB asset store that this could be copied to? Or perhaps another very large example FGB to show off?

Reply:
Also, it'd be more impressive if it was hosted behind a proper CDN.

Reply:
Don't have anything at the moment but I'll see what I can find.

Reply:
And here it is :)

http://flatgeobuf.septima.dk/population_areas.fgb

Reply:
done, thanks!